examples/00_quick_start/rbm_movielens.ipynb

```diff
@@ -60,7 +60,7 @@
       "System version: 3.7.11 (default, Jul 27 2021, 14:32:16) \n",
       "[GCC 7.5.0]\n",
       "Pandas version: 1.3.4\n",
-      "Tensorflow version: 2.6.1\n"
+      "Tensorflow version: 1.15.5\n"
```

Comment:
hey @pradjoshi can you merge the latest stating to your branch, reinstall using TF 2 and rereun again?

Reply:
hi @miguelgfierro I have reinstalled recommenders using TF2 and rerun the notebooks.